src/sdl3_syms.h

```diff
@@ -429,7 +432,7 @@ SDL3_SYM_RENAMED_RETCODE(bool,HapticPause,PauseHaptic,(SDL_Haptic *a),(a),return
 SDL3_SYM_RENAMED_RETCODE(bool,HapticRumbleInit,InitHapticRumble,(SDL_Haptic *a),(a),return)
 SDL3_SYM_RENAMED_RETCODE(bool,HapticRumblePlay,PlayHapticRumble,(SDL_Haptic *a, float b, Uint32 c),(a,b,c),return)
 SDL3_SYM_RENAMED_RETCODE(bool,HapticRumbleStop,StopHapticRumble,(SDL_Haptic *a),(a),return)
-SDL3_SYM_PASSTHROUGH_RETCODE(bool,HapticRumbleSupported,(SDL_Haptic *a),(a),return)
+SDL3_SYM_PASSTHROUGH_INT_BOOL(bool,HapticRumbleSupported,(SDL_Haptic *a),(a),return)
```

Comment:
How did this not error before?

Reply:
[On SDL2, possible return values are -1, SDL_TRUE and SDL_FALSE](https://github.com/libsdl-org/SDL/blob/0efb7c78294b95d62bda35d4cd18945ee578b4d8/src/haptic/SDL_haptic.c#L706C1-L714C2)
[On SDL3, possible return values are true and false](https://github.com/libsdl-org/SDL/blob/0be5b8070d5f1cb501b47f1a5d7c743e5535a883/src/haptic/SDL_haptic.c#L604C1-L611C1)

Reply:
SDL3_SYM_PASSTHROUGH_RETCODE matches type but returns 0 and -1, which flips the outcome

Reply:
This should be changed to a wrapper function rather than adding a new passthrough macro.

Reply:
Implemented SDL_HapticRumbleSupported as a wrapper function instead

Reply:
> SDL3_SYM_PASSTHROUGH_RETCODE matches type but returns 0 and -1, which flips the outcome

I wonder whether there are any other places where we screwed up like that. (I hope not..)